pandas/core/internals/blocks.py

```diff
@@ -2138,6 +2123,13 @@ def get_values(self, dtype=None):
             return result.reshape(self.values.shape)
         return self.values
 
+    def internal_values(self):
+        return self._ea_values
+
+    @cache_readonly
+    def _ea_values(self):
```

Comment:
hmm forgot about this one, yet another 'values' thing.

Reply:
this is private, just so we have the thing cached instead of re-constructing

Reply:
(or could not-cache and have one fewer 'values' thing)